WikiStuff.py

```diff
@@ -0,0 +1,123 @@
+from bs4 import BeautifulSoup
+from sphinx.util import requests
+
+HEADERS_GET = {
+    'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; WOW64; rv:49.0) Gecko/20100101 Firefox/49.0',
+    # 'User-Agent': 'Mozilla/5.0 (Linux; Android 7.0; SAMSUNG SM-G950F Build/NRD90M) AppleWebKit/537.36 (KHTML, like '
+    #             'Gecko) SamsungBrowser/5.2 Chrome/51.0.2704.106 Mobile Safari/537.36',
+    'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8',
+    'Accept-Language': 'en-US,en;q=0.5',
+    'Accept-Encoding': 'gzip, deflate',
+    'DNT': '1',
+    'Connection': 'keep-alive',
+    'Upgrade-Insecure-Requests': '1'
+}
+
+WIKI_SEARCH_RESULT_ATTRS = {"class": "mw-search-result-heading"}
+WIKI_SEARCH_RESULT_DESCRIPTION_ATTRS = {"class": "searchresult"}
+WIKI_TABLE_ATTRS1 = {"style": "background-color:#fff"}
+WIKI_TABLE_ATTRS2 = {"style": "background-color:#f7f7f7"}
+
+
+def is_number(n):
+    try:
+        float(n)  # Type-casting the string to `float`.
+        # If string is not a valid `float`,
+        # it'll raise `ValueError` exception
+    except ValueError:
+        return False
+    return True
+
+
+def findWikiUrl(album, artist):
+    found = False
+    query = "{title} {art}".format(art=artist, title=album).replace(" ", "+").replace("&", "and")
```

Comment:
the & with and is very nice! but you should replace it with %26 instead (if they called it &, they want it to be that way in there artist/album name as well